tests/Builders/ServersBuilderTest.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Vyuldashev\LaravelOpenApi\Tests\Builders;
+
+use Vyuldashev\LaravelOpenApi\Builders\ServersBuilder;
+use Vyuldashev\LaravelOpenApi\Tests\TestCase;
+
+class ServersBuilderTest extends TestCase
+{
+    /**
+     * @dataProvider providerBuild
+     * @param array $config
+     * @param array $expected
+     * @return void
+     */
+    public function testBuild(array $config, array $expected): void
+    {
+        $SUT = new ServersBuilder();
+        $servers = $SUT->build($config);
+        $this->assertSameAssociativeArray($expected[0], $servers[0]->toArray());
+        var_dump($servers[0]->toArray());
```

Comment:
Is this `var_dump` a remainder of testing/development? Should probably not end up here.

Reply:
Thank you. I forgot.